### ISSUE TITLE
Bypass "Loading Dashboard" screen when sharing report

### ISSUE THREAD

**Issue by meowsome** [neo4j-labs/neodash#112]
Hi,

When using the "Share Dashboard" feature and sharing a URL, it prompts the user for if they want to override their connecting, saying the following:

```
You are loading a Neo4j dashboard.
You will be connected to <neo4j connection address>.

This will override your current dashboard (if any). Continue?
```

After they click "Continue" at the bottom right, they are allowed to see the dashboard. Is there anyway to specify a parameter when sharing a dashboard via URL to bypass this screen and just immediately load the dashboard?

Thanks!

**Comment by parraguezr** [comment]
Agree with @meowsome it would be great to have such an option. The warning confuses the user of the dashboard and requires an additional explanation everytime a dashboard is shared.

**Comment by brainwave** [comment]
I am trying to implement this feature, but the  confusing part (new to React here) - how to skip past the LoadSharedDashboardModal? It is invoking a method onConfirmLoadSharedDashboard. What I am unsure of is: how can I trigger a call to this method "without" returning a modal component, if a url paramter is set?

I got as far as adding a paramter to the URL, parsing it, and adding it to the redux state such that the Application and LoadSharedDashboardModal - both have access to it.

Open to any implementation ideas / guidance on how to proceed further!

**Comment by brainwave** [comment]
I made some progress: `const continueButtonRef = useRef(null);

  useEffect(() => {
    if(continueButtonRef.current)
    continueButtonRef.current.click();
  });`

And the button was modified as such: `<Button
              component='label'
              id='test'
              ref = {continueButtonRef}
              onClick={() => {
                onConfirmLoadSharedDashboard();
              }}
              style={{ backgroundColor: 'white', marginTop: '20px', float: 'right' }}
              color='default'
              variant='contained'
              endIcon={<PlayArrow />}
              size='medium'
            >`

Idea is that I can simply "press" continue on loading the LoadSharedDashboardModal. However, now I am getting the "Unable to load shared dashboard" message while the dashboard is successfully loading in the background. The trace says that its due unknown element "id". Could it be because I am programmatically clicking and not from the UI? I think if this one error is solved at least a workable (maybe not good) solution will be available for this requirement.

**Comment by brainwave** [comment]
I have created a PR (unfortunately the diff got botched due to another change I had made for internal use) - https://github.com/neo4j-labs/neodash/pull/436 

The PR needs to be cleaned up with a cherry pick only for this change but not sure how that can be done on github. In addition, the final way that worked was:
1. Conditionally render WelcomeScreenModal in application.tsx (otherwise the welcome screen shows up and doesn't dissappear)
2. In LoadSharedDashboardModal, if skipConfirmation url parameter has been passed, directly call subsequent dispatch function onLoadSharedDashboardConfirmation. That way, the confirmation screen is skipped if and only if this parameter is specified and set to true.

**Comment by brainwave** [comment]
I am sure the pr submitted is of crappy quality (even other than the extraneous files showing in . Specifically, the flag setWelcomeScreenOpen is mandatorily set to false, which I am sure is not the right approach. Unfortunately, when I tried setting it conditionally using urlParams, the modal still opened. I think I need some help from the community or dev team to fix it properly. Eagerly awaiting PR Review comments to improve this code.

**Comment by brainwave** [comment]
Hi : I have submitted a new PR - https://github.com/neo4j-labs/neodash/pull/439 with code that actually works. Its not clean yet (will clean it up tomorrow) but at least the code is working properly now. Waiting for tests to pass.

**Comment by brainwave** [comment]
Hi all, I have created a new PR for this feature. The code change is not idea (needs a setTimeout to work, must be a race condition somewhere). 

https://github.com/neo4j-labs/neodash/pull/443 

Open to ideas on how I can improve the PR.

**Comment by alfredorubin96** [comment]
I'll close this issue because it has been fixed in #476

**Comment by bb-xops** [comment]
how do i activate this ? I see that it says that it has been fixed but how does one actually implement it in the code ?
